src/components/tx/ConfirmationOrder/TokenInfoPair/index.tsx

```diff
@@ -0,0 +1,77 @@
+import { Stack, Box, Typography, SvgIcon } from '@mui/material'
+import EastRoundedIcon from '@mui/icons-material/EastRounded'
+import TokenIcon from '@/components/common/TokenIcon'
+import TokenAmount from '@/components/common/TokenAmount'
+
+export type InfoBlock = {
+  value: string
+  label: string
+  tokenInfo?: {
+    decimals: number
+    symbol: string
+    logoUri?: string | null
+  }
+}
+
+const TokenInfoPair = ({ blocks, showArrow }: { blocks: [InfoBlock, InfoBlock]; showArrow?: boolean }) => {
```

Comment:
That name is a small lie & you know it :)
Maybe a better name would be BlockInfoPair?

Reply:
Maybe `ConfirmationOrderHeader`?